### DISCUSSION TITLE
Which model of Raspberry PI are you using with Kairos?

### DISCUSSION THREAD

**Discussion by mudler** [kairos-io/kairos#1608] [Polls]
Hi everyone,

background: we are looking to streamline support for Raspberry PI 4 and fade out support to Raspberry PI 3, so we are wondering how many users out there are still using RPI3, and if it makes sense to invest time in supporting both platforms.

**Comment by tantalic** [comment]
I'm currently using a mix of both, because I have some older Pis around and getting a new PI 4 has been pretty challenging over couple of years. However, that seems to be changing and PI 3 will only get less common/useful over time so it might make sense to "skate where the puck is going" on this one at this time, rather than investing in a platform who's use will only go down.

**Comment by jbalonso** [comment]
I'm actually using both: my cluster servers are RPi4, and I have a sprinkler controller that's an RPi3. (Full disclosure: I haven't joined the sprinkler controller yet, but I have been planning to).

**Comment by RyzeNGrind** [comment]
Perhaps it may not hurt to have worker node support for arm32 while focusing on arm64 mgmt and worker planes. I currently mostly use RPI4 (arm64) in my cluster but I do have a RPI3 (arm32) that I use here and there with the cluster.
